dot/node.go

```diff
@@ -333,19 +386,52 @@ func NewNode(cfg *Config, ks *keystore.GlobalKeystore) (*Node, error) {
 
 	telemetry.GetInstance().AddConnections(telemetryEndpoints)
 	genesisHash := stateSrvc.Block.GenesisHash()
+	peerID := ""
+	if networkSrvc != nil {
+		peerID = networkSrvc.NetworkState().PeerID
+	}
 	err = telemetry.GetInstance().SendMessage(telemetry.NewSystemConnectedTM(
 		cfg.Core.GrandpaAuthority,
 		sysSrvc.ChainName(),
 		&genesisHash,
 		sysSrvc.SystemName(),
 		cfg.Global.Name,
-		networkSrvc.NetworkState().PeerID,
+		peerID,
 		strconv.FormatInt(time.Now().UnixNano(), 10),
 		sysSrvc.SystemVersion()))
 	if err != nil {
 		logger.Debugf("problem sending system.connected telemetry message: %s", err)
 	}
-	return node, nil
+}
+
+//InitKeystore to initialize node keystore
+func InitKeystore(cfg *Config) (*keystore.GlobalKeystore, error) {
```

Comment:
couldn't find any reference to this method.

Reply:
It's used in some tests, I've moved it to the test file, and made it private.

Reply:
@edwardmack was looks like this wasn't moved?

Reply:
@noot opps, looks like I did move it and make it private, but didn't remove this one.  It's gone now.